src/Appwrite/Extend/Exception.php

```diff
@@ -235,6 +235,8 @@ class Exception extends \Exception
 
     /** Health */
     public const QUEUE_SIZE_EXCEEDED                 = 'queue_size_exceeded';
```

Comment:
While we are at it can we also update the key here to `HEALTH_QUEUE_SIZE_EXCEEDED`